Feat ar61 and 7

urkle opened the pull request.

urkle committed:
remove schema_plus_compatibility dependency

- this is no longer needed as AR 5.2+ already filters views out of the table list

urkle committed:
add in AR 6.1 and 7.0 support, also Ruby 3.1

felipemaier reviewed (approved):
Might worth mentioning that you are removing the dependency of schema_plus_compatibility  in the "release notes" section.

urkle committed:
release: 1.0.1

- Add AR 6.1 & 7.0
- Add Ruby 3.1
- remove dependency on schema_plus_compatibility
